### ISSUE TITLE
Maybe a mistake in the README.md ?

### ISSUE THREAD

**Issue by mingodad** [eatkins/tree-sitter-ebnf-generator#4]
Looking at the `README.md` and the explanation/example shown bellow:
```
    matches either the expression to the left of the | or the expression
    to the right. Concatenation has higher precedence than alternation
    so (A B) | (C | D) is equivalent to A B | C D.
```
If I understood it correctly because we are using grouping `()` to override precedence the example above could not be equivalent.
```
'(A B) | (C | D)' <> 'A B | C D'
```

**Comment by mingodad** [comment]
And this is one of the reasons I'm advocating to have an option to output an **EBNF** compatible with  https://www.bottlecaps.de/rr/ui to help visually understand the iterations of a rule.

Input on **Edit Grammar** tab:
```
equiv ::= (A B) | (C | D)
```
Simplified shown on **View Diagram** tab:
```
equiv    ::= A B
           | C
           | D
```
And here in https://arthursonzogni.com/Diagon/#Grammar (as their **EBNF** syntax):
```
equiv = (A , B) | (C | D);
```
Output:
```
equiv:
    │├──╮── A ── B ──╭──┤│
        │            │
        ╰──── C ─────╯
        │            │
        ╰──── D ─────╯
```

**Comment by eatkins** [comment]
You are correct. There was a typo in the README.

**Comment by eatkins** [comment]
Fixed by #6
